clients/translate/lib/google_api/translate/v2/api/detections.ex

```diff
@@ -55,19 +55,19 @@ defmodule GoogleApi.Translate.V2.Api.Detections do
           {:ok, GoogleApi.Translate.V2.Model.DetectionsListResponse.t()} | {:error, Tesla.Env.t()}
   def language_detections_detect(connection, opts \\ []) do
     optional_params = %{
-      :oauth_token => :query,
-      :bearer_token => :query,
       :upload_protocol => :query,
       :prettyPrint => :query,
       :uploadType => :query,
       :fields => :query,
-      :callback => :query,
       :"$.xgafv" => :query,
+      :callback => :query,
```

Comment:
What governs this param ordering? Why is it getting reordered?

Reply:
This is happening in the java code in swagger-cli. It's possible to fix upstream via sorting.